rpcs3/Emu/Cell/PPULLVMRecompiler.h

```diff
@@ -8,12 +8,12 @@
 #include "Emu/Cell/PPUDecoder.h"
 #include "Emu/Cell/PPUThread.h"
 #include "Emu/Cell/PPUInterpreter.h"
-#include "llvm/ExecutionEngine/ExecutionEngine.h"

```

Comment:
Not sure this is a "good" thing to rely on implicitly included files in other (foreign) headers.